packages/components/src/MultiSelect/MultiSelect.tsx

```diff
@@ -76,18 +84,27 @@ export const MultiSelect = ({
           ref={toggleButtonRef}
           id={`${id}--toggle`}
           aria-labelledby={`${id}--label`}
+          aria-describedby={validationId}
```

Comment:
This will probably conflict with Dug's PR so may need to loop back here if that gets merged first